python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -473,3 +474,31 @@ def _init_tvt_set(
                 )
             ret = ItemSetDict(data)
         return ret
+
+
+class BuiltinOnDiskDataset(OnDiskDataset):
```

Comment:
BuiltinDataset should be good, I can't image a BuiltinDataset which is not OnDiskDataset.
```suggestion
class BuiltinDataset(OnDiskDataset):
```